### PR TITLE
refactor: use css modules for components [TECH-337]

### PR DESCRIPTION
**Pull request by jenniferarnesen**
No functional changes

Changes include:
* replace `<Fragment>` with `<>`
* port css to css modules

Updating the app to follow the css standard determined here: https://github.com/dhis2/notes/issues/79#issuecomment-563933328

Fixes: https://jira.dhis2.org/browse/TECH-337

### CODE REVIEW

File: src/components/TitleBar/ViewTitleBar.js [hunk 1]
```diff
@@ -22,40 +21,10 @@ import {
...
-        alignItems: 'flex-start',
-    },
-    titleBarIcon: {
-        marginLeft: 5,
-        position: 'relative',
-        top: 1,
-        cursor: 'pointer',
-    },
-};
+const NO_DESCRIPTION = i18n.t('No description');
```
Comment (martinkrulltott) [inline src/components/TitleBar/ViewTitleBar.js:27]: Seems like this is only used once, should it still be a const?

File: src/components/TitleBar/ViewTitleBar.js [hunk 2]
```diff
@@ -92,15 +60,15 @@ class ViewTitleBar extends Component {
...
+            <>
                 <div className={classes.titleBar}>
                     <span style={titleStyle}>{name}</span>
                     <div className={classes.actions}>
                         <div
                             className={classes.titleBarIcon}
                             onClick={onStarClick}
                         >
-                            <StarIcon className={classes.starIcon} />
+                            <StarIcon style={{ fill: colors.grey600 }} />
```
Comment (martinkrulltott) [inline src/components/TitleBar/ViewTitleBar.js:71]: className instead of inline styling?

**Review by martinkrulltott — APPROVED** [review]
LGTM, just two tiny comments

File: src/components/TitleBar/ViewTitleBar.js (hunk 2 above)
Comment (jenniferarnesen) [reply src/components/TitleBar/ViewTitleBar.js:71]: I had to do this one with the style property due to css priority - MUI was overriding the color. Will get rid of this as soon as we migrate away from MUI icons

File: src/components/TitleBar/ViewTitleBar.js (hunk 1 above)
Comment (jenniferarnesen) [reply src/components/TitleBar/ViewTitleBar.js:27]: Good point

**Pull request merged**